doc/api/deprecations.md

```diff
@@ -429,12 +429,15 @@ changes:
   - version: v7.0.0
     pr-url: https://github.com/nodejs/node/pull/8217
     description: Runtime deprecation.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/35316
+    description: End-of-Life.
 -->
 
-Type: Runtime
+Type: End-of-Life
 
-Unhandled promise rejections are deprecated. In the future, promise rejections
-that are not handled will terminate the Node.js process with a non-zero exit
+Unhandled promise rejections are deprecated. Promise rejections
+that are not handled terminate the Node.js process with a non-zero exit
```

Comment:
It might be good to discuss the command line flag to alter behavior here. At the very least, include a reference to the relevant docs

Reply:
OK, added a sentence that includes a link to the CLI flag. PTAL.